### ISSUE TITLE
Broken link between Smart NORA cloud and local Node-Red instance.

### ISSUE THREAD

**Issue by qeggs** [andrei-tatar/node-red-contrib-smartnora#171]
The link is broken between the Smart NORA cloud service and my Node Red instance.

I have confirmed that Google Home is still connected to Smart NORA, because changing a switch in the Google Home App is properly reflected in the Smart Nora.

Everything worked properly 12 hours ago, but stopped working overnight. I have made no changes to my network configuration or node-red configuration. I have confirmed node-red has the proper configuration.

My internet connection occasionally drops for seconds or minutes, but it's working now. Would a temporary drop in internet access cause a permanent issue with the connection from Smart NORA cloud to my local node-red instance?

My user id is: VjV8zrrw3KPfE3SrA2PxN3AJ6Hz2

**Comment by qeggs** [comment]
Solved my issue by restarting the raspberry pi which was running my node-red instance.

**Comment by andrei-tatar** [comment]
@qeggs  There were some issues in previous versions where a network connectivity issue would cause the connection to be lost and not retried.
Please upgrade the package to the latest version for the best stability and performance.

**Comment by qeggs** [comment]
I updated to the latest package and now my nora nodes are missing from the palette. What gives? I restarted the docker container which contains node-red after the upgrade, but maybe this wasn't the right thing to do?

**Comment by andrei-tatar** [comment]
@qeggs any logs from node-red?
Usually docker containers don't persist changes between restarts unless you mount a writable volume. 
Did you have any other nodes disappear?

**Comment by qeggs** [comment]
I'm pretty unfamiliar with what's going on behind the scenes, so I'm not sure how to retrieve logs to share. I followed a guide to create a raspberry pi server which hosts MQTT and Node-red. I must have created a writable volume since the system is unchanged after restarting the pi.

No other nodes disappeared. Everything works except Smart NORA. Seems like I should be able to remove and reinstall Smart NORA to regain the functionality, but I don't see an "uninstall" option. It shows to latest version is installed, but there are no nodes.

I apologize for not knowing the important details of how my system works. I'm happy to go read anything that you may suggest to increase my understanding.

**Comment by andrei-tatar** [comment]
@qeggs can you share the guide you followed for setting up node-red in docker?
Usually connecting to the docker container and running `journalctl -u nodered.service` should provide the logs. 

I'm running node-red directly on rpi (not via docker) and I just:
![image](https://user-images.githubusercontent.com/6473183/162770057-cb30abc6-99aa-4bf9-8918-eb65612b12b3.png)

**Comment by andrei-tatar** [comment]
@qeggs as documented in the link, you can try starting docker with `docker run -it -p 1880:1880 -v node_red_data:/data --name mynodered nodered/node-red`. This will attach a terminal and show the logs. After that, just copy the logs over here.

**Comment by qeggs** [comment]
I've resolved the issue by updating node-red to the latest version! To do that, I followed instructions here [https://nodered.org/docs/getting-started/docker](https://nodered.org/docs/getting-started/docker)

There was a hiccup after the update where I couldn't edit my smartnora configuration node. I had to "enable" that specific node within the palette manager, which allowed me to create a new configuration.
![image](https://user-images.githubusercontent.com/52592447/162874833-0d4bb88f-1c81-4e07-93d9-1cd7d2210909.png)
